clients/ts/signalr/src/LongPollingTransport.ts

```diff
@@ -147,6 +147,14 @@ export class LongPollingTransport implements ITransport {
                 }
             }
         } finally {
+            // Mark the poll aborted.
+            this.pollAbort.abort();
+
+            // If we were on the shutdown timer, stop it.
+            if (this.shutdownTimer) {
+                clearTimeout(this.shutdownTimer);
+            }
```

Comment:
I noticed this bug while running the tests and can fork it off into a separate PR if we don't take this for RTM. We need to mark the poll "aborted" when we shutdown normally so we don't start the shutdownTimer. And, if the shutdownTimer is running, we need to terminate it because we have shut down.

This does not cause a functional issue, but it manifested because Jest waits for all timers to elapse before shutting down. You'll see this manifest as a pause after Jest says the test suite has finished but before it actually terminates. It's useful that it does that, because we'd otherwise miss things like this.

Reply:
For visibility I think it should be a separate PR anyway.

Reply:
You sound like my conscience :). Yeah, that's the right thing to do.